philip/src/components/Publish.svelte

```diff
@@ -26,6 +29,7 @@
       .map(getHashTag);
 
     const data = wrapHashTagsWithLink(content);
+    const data = wrapHashTagsWithLink(wrapLinksWithTags(content));
```

Comment:
Looks like only should be one `data` 
`(plugin svelte) ParseError: Identifier 'data' has already been declared`